freckle-memcached/library/Freckle/App/Memcached/Client.hs

```diff
@@ -56,14 +64,15 @@ get
   => CacheKey
   -> m (Maybe Value)
 get k = traced $ with $ \case
-  MemcachedClient mc -> liftIO $ view _1 <$$> Memcache.get mc (fromCacheKey k)
```

Comment:
Curious if you have an opinion on `<$$>`. I like the idea of it being in `Freckle.App.Prelude` and use it when available, but when I'm not already using a prelude, I rarely define or import it.

Reply:
I don't like it; I hardly even like `<$>` most of the time.